.woodpecker.star

```diff
@@ -334,13 +332,47 @@ def build(ctx):
 
     return pipelines
 
+def release_ready_go():
```

Comment:
Why are you naming it `release_ready_go` when the plugin name seems to be `ready_release_go`? 😁

Reply:
fixed